lib/swagger_client/version.rb

```diff
@@ -3,13 +3,13 @@
 
 #No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
 
-OpenAPI spec version: 1.0.28
+OpenAPI spec version: 1.0.29
 
 Generated by: https://github.com/swagger-api/swagger-codegen.git
-Swagger Codegen version: 2.4.19
+Swagger Codegen version: 2.4.27
 
 =end
 
 module SwaggerClient
-  VERSION = '1.0.28'
+  VERSION = '1.0.0'
```

Comment:
The script automatically wipes this version number. We have to manually update it

Reply:
Good catch, and that is annoying.